config/database.yml

```diff
@@ -24,10 +24,16 @@ default: &default
 development:
   <<: *default
   database: smogmapper_development
+  username: <%= ENV['SMOGMAPPER_DB_USERNAME'] %>
+  password: <%= ENV['SMOGMAPPER_DB_PASSWORD'] %>
+  host: <%= ENV['SMOGMAPPER_DB_HOST'] %>
```

Comment:
This is required for docker since postgresql is started in separate container. During normal development nothing has to be changed (`ENV['non_existing_ley']` will return null).